src/visualizer/seaport/index.ts

```diff
@@ -0,0 +1,166 @@
+import { PROTOCOL_ID } from "..";
+import { ASSET_TYPE, AssetInOut, Domain, Protocol, Result } from "../../types";
+import { SeaPortItemType, SeaPortPayload } from "../../types/seaport";
+import { isSameAddress } from "../../utils";
+
+export const isCorrectDomain = (domain: Domain) => {
+  return (
+    supportedChains.includes(Number(domain.chainId)) &&
+    addressesBook.includes(domain.verifyingContract.toLocaleLowerCase())
+  );
+};
+
+export const visualize = (message: SeaPortPayload, domain: Domain): Result => {
+  if (!isCorrectDomain(domain)) throw new Error("wrong domain");
```

Comment:
In the current implementation, `supportedChains` is `[1, 2, 4, 5]` which is Mainnet, and it's testnet.
Would be worth adding all chains supported by Opensea for broader support, here is the [link](https://support.opensea.io/hc/en-us/articles/4404027708051-Which-blockchains-are-compatible-with-OpenSea-)

Reply:
nice catch